web-ui/src/main/resources/catalog/components/viewer/wfs/partials/download.html

```diff
@@ -1,7 +1,7 @@
 <p data-ng-show="formats.length > 0 && featureType" data-translate=""
    wfsDownloadDataInstruction
 </p>
-<div data-ng-hide="isWfsAvailable">
+<div data-ng-hide="isWfsAvailable && formats.length > 0 && !featureType">
```

Comment:
i noticed that sometimes iswfsavailable is true, but the formats list is empty, the code would then fail later because it can not find relevant format, better to detect in advance (or extend the iswfsavailable evaluation to include formats.length>0

Reply:
let me verify about !featuretype, maybe it relates to hiding the element in case a featuretype is already selected

Reply:
A bit weird that this url is fine
http://services.inspire-provincies.nl/ProductionAndIndustrialFacilities/services/download_PF?request=getcapabilities&service=wfs
but the same url via geonetwork proxy throws the required uppercase WFS error
a workaround is to educate people to use uppercase WFS when calling deegree webservices
but at some point gn could enforce uppercase WFS

Reply:
Alright so let's settle on assuming gml 3.2 as a default format when none was provided, and see what is the response on your issue on deegree.

As for the `!featureType` condition I'm still not sure I get it... is it really needed? Semantically, the error should simply appear when the WFS is not available.

Reply:
i guess formats.length>0 is now covered by the default format
!featuretype is indeed unclear, removed it in latest commit